<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1921</width>
    <height>917</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow { background-image: url(:/Images/Chalkboard.png);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="DictionaryLabel">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Images/Clipboard.png);
background-color: transparent;
background-image:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="DictionaryWords">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
background-image:none;</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Add Dictionary words here...</string>
    </property>
   </widget>
   <widget class="QLabel" name="PassageLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>731</width>
      <height>791</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Images/Board.png);
background-color: transparent;
background-image:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="PassageWords">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>150</y>
      <width>601</width>
      <height>551</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Insert passage to spellcheck here...</string>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorLogLabel">
    <property name="geometry">
     <rect>
      <x>-140</x>
      <y>520</y>
      <width>951</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Images/Error Log.png);
background-color: transparent;
background-image:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Spellcheck!</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ErrorLogWords">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>540</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: transparent;
background-image:none;</string>
    </property>
    <property name="placeholderText">
     <string>Misspelled words will appear here...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1921</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
